src/main/java/net/dv8tion/jda/api/events/channel/priv/PrivateChannelCreateEvent.java

```diff
@@ -26,7 +27,11 @@
  * Indicates that a {@link net.dv8tion.jda.api.entities.PrivateChannel Private Channel} was created.
  *
  * <p>Can be used to retrieve the created private channel and its {@link net.dv8tion.jda.api.entities.User User}.
+ *
+ * @deprecated This event is no longer supported by discord
  */
+@ForRemoval(deadline="4.3.0")
```

Comment:
Probably should change this deadline to
```suggestion
@ForRemoval(deadline="4.4.0")
```